### PR TITLE
[stable27] fix(install): Make installing more verbose

### PR DESCRIPTION
**Pull request by nickvergessen**
Backport of #41214

### CODE REVIEW

File: lib/public/Migration/IOutput.php [hunk 1]
```diff
@@ -29,6 +29,13 @@
  * @since 9.1.0
  */
 interface IOutput {
+	/**
+	 * @param string $message
+	 * @return void
+	 * @since 28.0.0
```
Comment (ChristophWurst) [inline lib/public/Migration/IOutput.php:35]: :crying_cat_face:

Comment (nickvergessen) [reply lib/public/Migration/IOutput.php:35]: Why are you sad, cat? :cuddle:

I guess we are at the point again where a differentiation between "interfaces to help apps what to expect" (adding things would be okay) vs. "interfaces that apps can implement to overwrite services" (adding things is less okay) would be helpful, and this here is the first case.

You want me to deprecate this interface and add a new one instead?

Comment (nickvergessen) [reply lib/public/Migration/IOutput.php:35]: Ah, it's the backport. Well this seems to be how we roll with this topic atm:
https://github.com/nextcloud/server/blob/stable26/lib/public/OCM/IOCMDiscoveryService.php#L45

But can also set it to 27.1.4

Comment (kesselb) [reply lib/public/Migration/IOutput.php:35]: Please 27.1.4 :pray:

**Review by skjnldsv — APPROVED** [review]

**Review by kesselb — APPROVED** [review]

**Pull request merged**